src/index.js

```diff
@@ -13,14 +13,14 @@ const root = reactDom.createRoot(document.getElementById("root"));
 
 root.render(
   <Provider store={store}>
-    <RoleProvider>
-      <RoleDetailsProvider>
-        <PermissionProvider>
+    {/* <RoleProvider> */}
+      {/* <RoleDetailsProvider> */}
+        {/* <PermissionProvider> */}
```

Comment:
Did you consult with @tjeaneric and @UmuhireJessie before commenting out roles & permissions-related code? if so please remove the commented-out code then

Reply:
I just returned them back the way they were.